content/docs/user-guide/project-structure/dvc-files.md

```diff
@@ -65,6 +66,7 @@ The following subfields may be present under `outs` entries:
 | `cache`                         | Whether or not this file or directory is <abbr>cached</abbr> (`true` by default). See the `--no-commit` option of `dvc add`.                                                                                                                                                                                                          |
 | `persist`                       | Whether the output file/dir should remain in place while `dvc repro` runs (`false` by default: outputs are deleted when `dvc repro` starts)                                                                                                                                                                                           |
 | `desc`                          | (Optional) user description for this output (supported in metrics and plots too). This doesn't affect any DVC operations.                                                                                                                                                                                                             |
+| `remote`                        | (Optional) name of the remote to use for pushing/fetching                                                                                                                                                                                                                                                                             |
```

Comment:
Just moving this above less interesting fields...
```suggestion
| `remote`                        | (Optional) name of the remote to use for pushing/fetching                                                                                                                                                                                                                                                                             |
| `persist`                       | Whether the output file/dir should remain in place while `dvc repro` runs (`false` by default: outputs are deleted when `dvc repro` starts)                                                                                                                                                                                           |
| `desc`                          | (Optional) user description for this output (supported in metrics and plots too). This doesn't affect any DVC operations.                                                                                                                                                                                                             |
```

Reply:
p.s. not sure we need the "(optional)" remarks. They're all optional I think. Maybe they should all indicate what's the default value instead. Out of scope though (but feel free to apply that rule to the one you added).